doc/building.md

```diff
@@ -633,6 +633,22 @@ libffi-devel`.
 Use `--with-libffi=<path>` if `configure` does not properly locate your libffi
 files.
 
+### libsleef
+
+libsleef, the [SIMD Library for Evaluating Elementary Functions](
+https://sleef.org/) is required when building libvmath.so on Linux/aarch64
```

Comment:
This is incorrect. The library is not required, but if it is present, we will build libvmath with it.

Reply:
Hi @magicus , the doc is updated. Thanks for your comment on this!